src/Tutorial/ui/DocumentationRoot.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from "react";
+
+import Button from "@mui/material/Button";
+import { MD } from "../../ui/md/md";
+
+import { getPage } from "./root";
+import { Navigator, useHistory } from "../../ui/React/Documentation";
+
+const resolveRelativePath = (folder: string, relative: string): string => {
+  const noLastSlash = folder.endsWith("/") ? folder.slice(0, folder.length - 1) : folder;
+  const lastIndex = noLastSlash.lastIndexOf("/");
+  const prefix = lastIndex === -1 ? "" : noLastSlash.slice(0, lastIndex + 1);
+  const suffix = relative.slice("../".length);
+  return prefix + suffix;
+};
+
+const resolvePath = (currentPath: string, newPath: string): string => {
+  const lastIndex = currentPath.lastIndexOf("/");
+  const folder = lastIndex === -1 ? "" : currentPath.slice(0, lastIndex + 1);
+  if (!newPath.startsWith("../")) return folder + newPath;
+
+  return resolveRelativePath(folder, newPath);
+};
```

Comment:
Adding imports and removing duplicate functionality at DocumentationRoot
```suggestion
import { isFilePath, resolveFilePath } from "../../Paths/FilePath";
```